Add correlation engine to Security Analytics documentation (#3814)
opensearch-project/documentation-website: _security-analytics/usage/correlation-graph.md

## Acccessing the graph

Begin by selecting **Security Analytics** in the Dashboards home page menu. Then select **Correlations** from the Security Analytics menu on the left side of the screen. The **Correlations** page is displayed.

<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph.png" alt="The correlation graph" width="85%">

## Interpreting the graph

The graph displays findings as nodes with colored borders expressing their severity level. A three-letter abbreviation inside the node indicates the log type. The lines that connect the findings represent the correlations between them. A heavy line indicates a strong correlation, while a light line shows a weaker connection.

> <img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph-detail.png" alt="The correlation graph" width="40%">

Reviewer: Reference the image in the text that precedes it.

## Using the graph

> You can control which findings are displayed in the graph by filtering by severity level, log type, and time filter. The time filter controls the findings that appear in the graph by limiting the dates on which the findings were generated.

Reviewer: "on" the graph (all instances in this section)? "by limiting the dates on which the findings were generated" is slightly unclear. Can we simplify?

> * Use the **Severity** dropdown list to select which findings appear in the graph by their severity level. The number beside the list name shows how many severity levels are being shown in the graph.

Reviewer: ```suggestion
* Use the **Severity** dropdown list to select which findings appear in the graph according to their severity level. The number beside the list name indicates how many severity levels are being shown in the graph.
```

> * Use the **Log types** dropdown list to select which log types to show in the graph. The number beside the list name shows how many log types are being shown in the graph.

Reviewer: ```suggestion
* Use the **Log types** dropdown list to select which log types to show in the graph. The number beside the list name indicates how many log types are being shown in the graph.
```

* Select **Reset filters** to return the dropdown lists to their default, showing all items.

> * Use the time filter to set the the date range and show only those findings that were generated within that time span. Select **Refresh** to bring the current number of findings up to date.

Reviewer: ```suggestion
* Use the time filter to set the date range and show only those findings that were generated within that time span. Select **Refresh** to bring the current number of findings up to date.
```

> You can drill down in the graph to look at correlations associated with a specific finding by selecting the finding in the graph. The graph then changes to show only the selected finding along with the constellation of findings correlated to it.

Reviewer: Let's use something other than "drill down," which may not be understood by all readers.

Author: Yes.
"You can focus on a particular area of the graph to look at correlations associated with a specific finding..."

<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-drill-dn.png" alt="Zooming in on a specific finding in the graph" width="40%">

> After drilling down in the graph, informational cards for each of the findings appear in the right-hand side of the screen. The selected finding appears first at the top of the cards and the correlated findings are listed below it in order of their correlation relevance, represented by a correlation score.

Reviewer: Same comment re: "drilling down."

> After drilling down in the graph, informational cards for each of the findings appear in the right-hand side of the screen. The selected finding appears first at the top of the cards and the correlated findings are listed below it in order of their correlation relevance, represented by a correlation score.

Reviewer: ```suggestion
After drilling down in the graph, informational cards for each of the findings appear on the right-hand side of the screen. The selected finding appears at the top of the cards, and the correlated findings are listed below it in order of their correlation relevance, represented by a correlation score.
```

> After drilling down in the graph, informational cards for each of the findings appear in the right-hand margin of the screen. The selected finding appears first at the top of the cards and the correlated findings are listed below it in order of their correlation relevance, represented by a correlation score.

Reviewer 2: In the **Correlation** panel? (instead of in the right-hand margin of the screen).

<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-cards.png" alt="Zooming in on a specific finding in the graph" width="30%">

> You can select one of the correlated findings in the graph to shift the perspective of the correlation relationships. This sends the newly selected finding first to the top of the informational cards and displays the other findings as relative correlations.

Reviewer: ```suggestion
You can select one of the correlated findings in the graph to shift the perspective of the correlation relationships. This sends the newly selected finding to the top of the informational cards and displays the other findings as relative correlations.
```
